mlos_bench/mlos_bench/config/schemas/environments/remote/saas-env-subschema.json

```
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/microsoft/MLOS/main/mlos_bench/mlos_bench/config/schemas/environments/remote/saas-env-subschema.json",
+    "title": "mlos_bench SaaSEnv config",
+    "description": "Config instance for a mlos_bench SaaSEnv",
+    "type": "object",
+    "$comment": "NOTE: We only list unique properties here. Others are inherited from the leaf-environment-subschema.json. Additionally, unevaluatedProperties handling is dealt with elsewhere.",
+    "properties": {
+        "class": {
+            "enum": [
+                "mlos_bench.environments.remote.SaaSEnv",
+                "mlos_bench.environments.remote.saas_env.SaaSEnv"
+            ]
+        },
+        "config": {
+            "type": "object",
+            "$comment": "NOTE: there aren't currently any additional properties accepted by the OSEnv.  In the future we may want to use an os_type enum to help automatic OS param tuning by default without RemoteEnv requirements.",
+            "allOf": [
+                {
+                    "$ref": "../common-environment-subschemas.json#/$defs/common_environment_config"
+                }
+            ],
+            "unevaluatedProperties": false
+        }
+    },
+    "required": ["class", "config"]
```

```suggestion
    "required": ["class"]
```

remove `config` if not actually required, else require at least one element.